fujiji-server/src/repositories/listing.js

```diff
@@ -35,6 +86,57 @@ async function getAllListingsByProvince(provinceCode) {
   }
 }
 
+async function getAllListingsByProvinceCategory(provinceCode, category) {
+  try {
+    const [listingsByProvinceCategory] = await sequelize.query(
+      'SELECT * FROM fujiji_listing WHERE province_code = ? AND category = ?',
+      {
+        replacements: [provinceCode, category],
+        type: Sequelize.SELECT,
+      },
+    );
+    console.log('DEBUG-listingsByProvinceCategory');
+    console.log(listingsByProvinceCategory);
+    return listingsByProvinceCategory;
+  } catch (err) {
+    return err;
+  }
+}
+
+async function getAllListingsByProvinceCondition(provinceCode, condition) {
+  try {
+    const [listingsByProvinceCondition] = await sequelize.query(
+      'SELECT * FROM fujiji_listing WHERE province_code = ? AND condition = ?',
+      {
+        replacements: [provinceCode, condition],
+        type: Sequelize.SELECT,
+      },
+    );
+    console.log('DEBUG-listingsByProvinceConditon');
+    console.log(listingsByProvinceCondition);
+    return listingsByProvinceCondition;
+  } catch (err) {
+    return err;
+  }
+}
+
+async function getAllListingsByProvincePriceRange(provinceCode, startPrice, endPrice) {
+  try {
+    const [listingsByProvincePriceRange] = await sequelize.query(
+      'SELECT * FROM fujiji_listing WHERE province_code = ? AND price BETWEEN ? AND ?',
+      {
+        replacements: [provinceCode, startPrice, endPrice],
+        type: Sequelize.SELECT,
+      },
+    );
+    console.log('DEBUG-listingsByProvincePriceRange');
+    console.log(listingsByProvincePriceRange);
```

Comment:
I think we can remove these `console.logs`. I just read sequelize have option that can be passed on for logging

Reply:
Ah got it!

Reply:
That only logs the sql query that we're doing (i.e., SELECT * ...), not the result that we got from the query, right? (at least that's what I see when trying it)

Reply:
Ah yes you're correct. Okay, then if that's the case, what do you think about refactoring the `console.log` into a separate logger function so we can re-use that function instead?

Reply:
Sure, that would be cleaner!